tests/roots/test-latex-table/expects/table_having_caption.tex

```diff
@@ -4,7 +4,7 @@
 \centering
 \sphinxcapstartof{table}
 \sphinxcaption{caption for table}\label{\detokenize{tabular:id1}}
-\sphinxaftercaption
+\sphinxaftertopcaption
```

Comment:
Again, I don't think this change of name is breaking. There was no reason to use custom template with `\sphinxaftercaption` elsewhere: if user has custom table template with caption at bottom, there was no reason to use `\sphinxaftercaption` at all.